src/views/workflow-actions/workflow-action-start-form/helpers/get-cron-fields-error.ts

```diff
@@ -0,0 +1,26 @@
+import { CRON_FIELD_ORDER } from '@/components/cron-schedule-input/cron-schedule-input.constants';
+import { cronValidate } from '@/utils/cron-validate/cron-validate';
+import { type CronData } from '@/utils/cron-validate/cron-validate.types';
+
+export const getCronFieldsError = (
+  cronString: string
+): Partial<Record<keyof CronData | 'general', string>> | null => {
```

Comment:
You can define this type somewhere (and potentially also use it in the modal-content file)